src/main/resources/schema.sql

```diff
@@ -11,7 +11,15 @@ DROP TABLE IF EXISTS hmhb_user;
 CREATE TABLE hmhb_user (
     id BIGSERIAL PRIMARY KEY,
     email TEXT NOT NULL UNIQUE,
-    is_admin BOOLEAN NOT NULL
+    admin BOOLEAN NOT NULL,
+    display_name TEXT,
```

Comment:
I'm pretty sure a few of these will always be provided (display name, image url, profile url), however, I'm not 100% sure, so I've left them all nullable.